5_4_A.py

```diff
@@ -0,0 +1,35 @@
+def interleave(*iterables):
+    """
+    Standard version
+    :param iterables: Unknown amount of iterables
+    :return: the list of separated members of the iterables
+    """
+    list_of_items = []
+    for iterable in iterables:
+        for item in iterable:
+            list_of_items.append(item)
+    return list_of_items
+
+
+def interleave_generator(*iterables):
+    """
+    Generator version
+    :param iterables: Unknown amount of iterables
+    :return: the separated members of the lists
+    """
+    for iterable in iterables:
+        for item in iterable:
+            yield item
+

```

Comment:
Please notice - both of your function wont merge the iterables as requested.
(for example ['a', 1, '!', 'b', 2, '@', 'c', 3, '#'])
Check it and fix it if needed.